src/ui/simple_ui/ask_hash.rs

```diff
@@ -190,3 +262,55 @@ enum Recoverable {
     AskAgain,
     Skip,
 }
+
+#[cfg(test)]
+mod tests {
+    use super::parse_hashfile;
+    use std::io::Cursor;
+
+    #[test]
+    fn parse_simple_hashfile() {
+        let mut cursor = Cursor::new(
+            "bceb3dded8935c1d3521c475a69ae557e082839b46d921c8b400524470b5c965  archlinux-2024.11.01-x86_64.iso"
+        );
+
+        assert_eq!(
+            parse_hashfile(&mut cursor, "archlinux-2024.11.01-x86_64.iso"),
+            base16::decode("bceb3dded8935c1d3521c475a69ae557e082839b46d921c8b400524470b5c965").ok()
```

Comment:
use `.unwrap()` in tests rather than `.ok()`
```suggestion
            parse_hashfile(&mut cursor, "archlinux-2024.11.01-x86_64.iso").unwrap(),
            base16::decode("bceb3dded8935c1d3521c475a69ae557e082839b46d921c8b400524470b5c965").unwrap()
```